.circleci/config.yml

```diff
@@ -183,15 +237,24 @@ workflows:
       - set_bitsrc_registry
       - install_npm_deps:
           requires:
-              - checkout_code
+            - checkout_code
       - build:
           requires:
             - install_npm_deps
       - unit_test:
           requires:
             - build
-      - e2e_test:
+      - lint_eslint:
+          requires:
+            - build
```

Comment:
you don't  really need the build to finish, it's better to do it before the build not after

Reply:
I know. That's how I started. It didn't work for some reason, so I just duplicated the 'unit-test' task.